docker-compose.yml

```diff
@@ -1,9 +1,19 @@
+# Docker Compose def for local development
 version: '3'
 services:
   web:
     build: .
     volumes:
-      - .:/upaya
+      # Mount specific sub-directories into image for local development
```

Comment:
I like forcing us to be more mindful about what files do get copied over into the image.

Reply:
I am not sure if this is still too much OR too little.  If the purpose of this docker-compose.yml is for local dev use that balance is best decided by the devs who use it.